client/my-sites/guided-transfer/host-credentials-page/pressable.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import {
+	Email,
+	Password,
+	CreateAccountTip,
+	SubmitSection,
+	WPOrgURL,
+} from './fields';
+
+class Pressable extends Component {
```

Comment:
I know you didn't introduce this but I think these `host-credentials-page` components could/should be stateless functions instead of classes. From what I've read React plans to add performance optimizations for stateless functions that we'll be able to take advantage of.